src/pages/NewDiveForm.js

```diff
@@ -128,13 +137,67 @@ export default function AddDive({ onCreate, locationInfos }) {
             maxLength="500"
           />
         </LastDivStyled>
+        <ImageUpload>
+          {image ? (
+            <>
+              <img src={image} alt="undefined" />
```

Comment:
Has undefined a deeper meaning here? Otherwise you could consider giving it a general description of a possibly rendered image.